iceberg/iceberg-handler/src/test/queries/positive/mv_iceberg_partitioned_orc.q

```diff
@@ -0,0 +1,16 @@
+-- MV data is stored by partitioned iceberg testing the existing Hive syntax (also used by native mv) to specify partition cols.
+--! qt:replace:/(\s+uuid\s+)\S+(\s*)/$1#Masked#$2/
+-- SORT_QUERY_RESULTS
+
+drop materialized view if exists mat1;
+drop table if exists tbl_ice;
+
+create table tbl_ice(a int, b string, c int) stored by iceberg stored as orc tblproperties ('format-version'='1');
```

Comment:
It would be useful to also have 1 test of partitioned MVs with format-version=2.

Reply:
added